CompetitionTemplate/src/competition/subsystems/arm/ArmSubsystem.java

```diff
@@ -38,9 +38,9 @@ public ArmSubsystem(WPIFactory factory, XPropertyManager propManager) {
         
         rightArmMotor = factory.getSpeedController(7);
         
-        upperLimitSwitch = factory.getDigitalInput(5);
-        lowerLimitSwitch = factory.getDigitalInput(6);
-        encoder = factory.getEncoder(1, 2);
+        upperLimitSwitch = factory.getDigitalInput(1);
+        lowerLimitSwitch = factory.getDigitalInput(2);
+        encoder = factory.getEncoder(4, 5);
```

Comment:
have we started an electronics contract again for this year? I think we tried putting it in the wiki last time?